packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -38,10 +36,32 @@ export class FabricBase extends React.Component<
     if (win) {
       this._disposables.push(on(win, 'mousedown', this._onMouseDown, true), on(win, 'keydown', this._onKeyDown, true));
     }
+
+    const classNames = this._getClassNamesHelper();
+    if (this.props.applyThemeToBody) {
+      const currentDoc: Document = getDocument(this._rootElement.current)!;
```

Comment:
Should avoid `!` and instead check the return value.

Reply:
done